### PR TITLE
WbGeometry objectView in frontend

### PR DESCRIPTION
**Pull request by stokkoff**
# Request Template Witsml Explorer
Thank you for contributing to WITSML Explorer!
Before submitting this PR, please fill in the following template.

## Fixes
This pull request fixes #we-452

## Description
_Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change._

Implemented objectview in frontend for wbGeometry

## Type of change
_List the types of change. Remove from the list any points which are not necessary_

* Bugfix
* New feature (non-breaking change which adds functionality)
* Enhancement of existing functionality

## Impacted Areas in Application
_List general components of the application that this PR will affect:_

*Frontend for the wbGeometry object

## Checklist:
_Please tick all the boxes or remove the ones that aren't needed and explain why_

*Communication*
* [x] PR is related to an issue
* [x] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [x] New code is covered by passing tests

## Further comments
_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc..._

### CODE REVIEW

File: Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx [hunk 1]
```diff
@@ -184,6 +205,12 @@ const WellboreItem = (props: WellboreItemProps): React.ReactElement => {
       />
       <TreeItem nodeId={rigGroupId} labelText={"Rigs"} onLabelClick={() => onSelectRigGroup(well, wellbore, rigGroupId)} onContextMenu={preventContextMenuPropagation} />
       <TreeItem nodeId={riskGroupId} labelText={"Risks"} onLabelClick={() => onSelectRiskGroup(well, wellbore, riskGroupId)} onContextMenu={preventContextMenuPropagation} />
+      <TreeItem
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx]: Can have the label as "WbGeometries" as we already call it "Trajectories" in the menu. Should also have the menu item in alphabetical order.

File: Src/WitsmlExplorer.Frontend/components/ContentViews/WbGeometrysListView.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  }, [selectedWellbore]);
+
+  const getTableData = () => {
+    return wbGeometrys.map((wbGeometry) => {
+      return {
+        ...wbGeometry,
+        ...wbGeometry.commonData,
+        id: wbGeometry.uid,
+        mdBottom: `${wbGeometry.mdBottom?.value?.toFixed(4) ?? ""} ${wbGeometry.mdBottom?.uom ?? ""}`,
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/ContentViews/WbGeometrysListView.tsx]: There is no need to include properties here that will not be shown in the content table

Comment (stokkoff) [reply Src/WitsmlExplorer.Frontend/components/ContentViews/WbGeometrysListView.tsx]: Unsure how much i can remove here. Can i remove everything but ...wbGeometry.commonData? For now i left two more lines but let me know

File: Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx [hunk 3]
```diff
@@ -223,6 +244,12 @@ const WellboreItem = (props: WellboreItemProps): React.ReactElement => {
               selected={selectedTubular && selectedTubular.uid === tubular.uid ? true : undefined}
             />
           ))}
+        <TreeItem
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx]: The wbGeometries item should not be nested under the tubular item

Comment (stokkoff) [reply Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx]: Should be fixed now

**Review by janburak — APPROVED** [review]

**Pull request merged**